Dockerfile.jammy

```diff
@@ -62,6 +62,8 @@ RUN apt-get update -qq && \
     unixodbc-dev \
     wget \
     zip && \
+    # https://github.com/rstudio/lucid-images-shiny/issues/311
```

Comment:
I'd probably remove this - it's a link to a private repository and this is a public repository.